components/Header.vue

```diff
@@ -22,11 +43,14 @@
 <script lang="ts">
 import Vue from 'vue'
 import { userSession } from '../utils/userSession'
+import { NOTES_FILE } from '../utils/constants'
 import Logo from './Logo.vue'
+import Dropdown from 'bp-vuejs-dropdown';
```

Comment:
We can implement a simple dropdown with pure CSS, like the dropdown menu on https://vuejs.org/

Reply:
i will try and let you know